src/plugins/intel_gpu/tests/test_cases/gemm_gpu_test.cpp

```diff
@@ -1350,6 +1350,152 @@ class GemmBaseTest : public ::testing::TestWithParam<gemm_params> {
+    static const int max_random = 200;
+
+    cldnn::engine& engine = get_test_engine();
+    topology topology_ocl;
+    topology topology_onednn;
+
+    ExecutionConfig config_ocl;
+    ExecutionConfig config_onednn;
+
+    float tolerance = 0.0f;
+
+    void SetUp() override {
+        config_ocl.set_property(ov::intel_gpu::optimize_data(true));
+        config_ocl.set_property(ov::intel_gpu::queue_type(QueueTypes::in_order));
+        if (engine.get_device_info().supports_immad) {
+            config_onednn.set_property(ov::intel_gpu::optimize_data(true));
+            config_onednn.set_property(ov::intel_gpu::queue_type(QueueTypes::in_order));
+        }
+    }
+
+    void execute(T& p) {
+        auto input0_prim = get_mem(get_input_layout(p, 0));
+        auto input1_prim = get_mem(get_input_layout(p, 1));
+
+        network network_ocl(engine, topology_ocl, config_ocl);
+        network network_onednn(engine, topology_onednn, config_onednn);
+
+        network_ocl.set_input_data("input0", input0_prim);
+        network_ocl.set_input_data("input1", input0_prim);
+        network_onednn.set_input_data("input0", input1_prim);
+        network_onednn.set_input_data("input1", input1_prim);
+
+        compare(network_ocl, network_onednn, p);
+    }
+
+    void compare(network& network_ocl, network& network_onednn, T& p) {
+        auto outputs_ocl = network_ocl.execute();
+        auto outputs_onednn = network_onednn.execute();
+
+        ASSERT_EQ(outputs_ocl.size(), outputs_onednn.size());
+        ASSERT_EQ(outputs_ocl.size(), size_t(1));
+
+        auto val_ocl = get_output_values_to_float(network_ocl, outputs_ocl.begin()->first);
+        auto val_onednn = get_output_values_to_float(network_onednn, outputs_onednn.begin()->first);
+
+        ASSERT_EQ(val_ocl.size(), val_onednn.size());
+
+        for (size_t i = 0; i < val_ocl.size(); i++) {
+            ASSERT_NEAR(val_ocl[i], val_onednn[i], tolerance)
+                << "tolerance = " << tolerance
+                << "\ni = " << i
+                << "\nocl[i] = " << val_ocl[i]
+                << "\nonednn[i] = " << val_onednn[i];
+        }
+    }
+
+    layout get_input_layout(T& p, int in_no) {
+        auto pad = p.pad;
+        std::vector<int> pad_ = { 0, 0, pad.spatial[0], pad.spatial[1] };
+        if (in_no == 0)
+            return layout{ p.data_type_in0, p.input_format, p.in_shapes.at(0), padding{ pad_ } };
+        else if (in_no == 1)
+            return layout{ p.data_type_in1, p.input_format, p.in_shapes.at(1), padding{ pad_ } };
+        else
+            return layout{ p.data_type_in2, p.input_format, p.in_shapes.at(2), padding{ pad_ } };
+    }
+
+    cldnn::memory::ptr get_mem(cldnn::layout l) {
+        auto prim = engine.allocate_memory(l);
+        tensor s = l.get_tensor();
+        if (l.data_type == data_types::bin) {
+            VF<int32_t> rnd_vec = generate_random_1d<int32_t>(s.count() / 32, min_random, max_random);
+            set_values(prim, rnd_vec);
```

Comment:
If this is implemented here(not common function), I think we don't need to add such code as it is not running...

Reply:
I'll remove it